src/arithmetic/algebraic_expression/utils.c

```diff
@@ -240,3 +295,10 @@ void _AlgebraicExpression_FetchOperands(AlgebraicExpression *exp, const GraphCon
 		break;
 	}
 }
+
+void _AlgebraicExpression_FetchOperands(AlgebraicExpression *exp, const GraphContext *gc,
+										Graph *g) {
```

Comment:
I dislike that this is a header-exposed function, propose that we rename to `AlgebraicExpression_FetchOperands`.